src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -965,7 +961,7 @@ private bool ConsumeStringNextSegment()
                         }
 
                     Done:
-                        _bytePositionInLine += sawNewLine ? leftOver + idx : leftOver + idx + 1;  // Add 1 for the end quote of the string.
+                        _bytePositionInLine++;  // Add 1 for the end quote of the string.
                         _consumed = idx + 1;    // Add 1 for the end quote of the string.
                         _totalConsumed += leftOver;
```

Comment:
I wouldn't change this for 3.0, but while looking to see if this was correct I think I learned that `_totalConsumed` is `_totalConsumedFromPreviousSegments`, e.g. doesn't contain at all the current segment/span's consumed count.

So: Future improvement: Consider renaming _totalConsumed to better indicate its purpose.

Reply:
Yep, makes sense.